remo/sdk.py

```diff
@@ -762,3 +764,54 @@ def view_annotation_stats(self, annotation_set_id: int):
 
     def _view(self, url, *args, **kwargs):
         return self.viewer.browse(self.api.url(url, *args, **kwargs))
+    
+    def generate_annotations_from_folders(self, path: str, output_filename: str):
+        """
+        Creates a CSV annotation file for image classification tasks, where images are stored in folders with names matching the labels of the images. The CSV file is saved in the same input directory where images are stored. 
+        Example of data structure for a dog / cat dataset: 
+              - source_folder
+                  - dog
+                     - img1.jpg
+                     - img2.jpg
+                     - ...
+                  - cat
+                     - img199.jpg
+                     - img200.jpg
+                     - ...
+
+
+        Args: 
+               path_to_data_folder: path to the source folder where data is stored
+               output_filename: filename of the output annotation file
+
+        Returns: 
+                CSV of Files: CSV
+                CSV of data encoding: CSV
+        """
+
+        def natural_sort(l): 
+            convert = lambda text: int(text) if text.isdigit() else text.lower() 
+            alphanum_key = lambda key: [ convert(c) for c in re.split('([0-9]+)', key) ] 
+            return sorted(l, key = alphanum_key)
+        def classes(path):
+            classes = [d.name for d in os.scandir(path) if d.is_dir()]
+            classes = natural_sort(classes)
+            class_to_idx = {cls_name: i for i, cls_name in enumerate(classes)}
+            return classes, class_to_idx
+
+        classes_list, mapping = classes(path)
+
+        im_dict = {}
+        im_dict["file_name"] = "class_name"
+        for class_name in classes_list:
+            im_list = os.listdir(path + "/" + class_name)
```

Comment:
@SreeHarshaNelaturu it's better to use `os.path.join()` to build the path

Reply:
Will make the change